test/truefi2/TrueCreditAgency.test.ts

```diff
@@ -67,6 +68,8 @@ describe('TrueCreditAgency', () => {
     await creditAgency.allowPool(tusdPool.address, true)
 
     await creditOracle.setScore(borrower.address, 255)
+    await creditOracle.setMaxBorrowerLimit(owner.address, parseEth(100_000_000))
```

Comment:
Is this ever used?

Reply:
It is, in tests we `borrow()` couple times on behalf of the `owner`.